### PR TITLE
removes opening new CE job slots

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

15/15 Chief Engineer slots have been taken.

## Why It's Good For The Game

bug fix

## Changelog

:cl:
fix: The HoP can no longer open CE job slots again.
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
This isn't a fix.

File: code/modules/modular_computers/file_system/programs/jobmanagement.dm [hunk 1]
```diff
@@ -14,13 +14,13 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
...
 
 	var/change_position_cooldown = 30
-	//Jobs you cannot open new positions for
+	///Jobs blacklisted from having their slots edited.
 	var/list/blacklisted = list(
 		JOB_CAPTAIN,
 		JOB_HEAD_OF_PERSONNEL,
 		JOB_HEAD_OF_SECURITY,
 		JOB_RESEARCH_DIRECTOR,
-		JOB_CHIEF_MEDICAL_OFFICER,
```
Comment (GoldenAlpharex) [inline code/modules/modular_computers/file_system/programs/jobmanagement.dm:23]: oops

**Comment by GoldenAlpharex** [comment]
> This isn't a fix.

it is, check the code

**Comment by JohnFulpWillard** [comment]
> This isn't a fix.

It is a fix, because it isn't intentional.

**Comment by Plejek** [comment]
man's malding over 15 ce slots

File: code/modules/modular_computers/file_system/programs/jobmanagement.dm (hunk 1 above)
Comment (JohnFulpWillard) [reply code/modules/modular_computers/file_system/programs/jobmanagement.dm:23]: it was actually my fault (which was why I put GBP no update), I caused it in https://github.com/tgstation/tgstation/pull/63357/files

**Comment by GuillaumePrata** [comment]
John doing an Ath, nerfing the HoP job and labelling it as a fix...

**Pull request merged**